terragrunt/aws/alert_compromise/functions/broadcast_alert_tests.py

```diff
@@ -0,0 +1,55 @@
+import pytest
+from unittest import mock
+from broadcast_alert import lambda_handler
+
+
+@mock.patch("broadcast_alert.boto3.client")
```

Comment:
Thank you for adding unit tests as well - I guess I should have probably done that before 😬

Reply:
All good!  Infinite work, finite time ⏳